src/app/node-data/template.html

```diff
@@ -50,6 +50,10 @@
       </mat-error>
     </mat-form-field>
 
+    <div class="quota-widget">
+      <ng-container *ngTemplateOutlet="quotaWidget"></ng-container>
```

Comment:
This `node-data` component is used both in machine deployment dialog and cluster wizard so are we going to show this quota widget in cluster wizard as well? Because in cluster wizard we already have quota widget on the same step where `node-data` is being used.

Reply:
The newly added widget is only visible on the dialog view, as it's needed to be passes as an `@Input` property